earth_enterprise/src/SConstruct

```diff
@@ -480,6 +480,10 @@ gee_version_number = env.get_open_gee_version().get_short()
 
 
 # Check if the version of the default compiler is at least 4.8:
+if not env['CXX']:
+    print "Please, install the default C++ compiler for your distribution."
+    Exit(1)
+
 version = opengee.c_compiler.get_cc_version(env['CXX'])
```

Comment:
Might I suggest

```
if env['CXX']:
    version = opengee.c_compiler.get_cc_version(env['CXX'])
else:
    version = []
```

since it's slightly easier to unpack?

Reply:
Go for it